proteome-rest/src/main/java/org/uniprot/api/proteome/service/GeneCentricSortClause.java

```diff
@@ -29,23 +33,24 @@ protected Sort createDefaultSort(boolean hasScore) {
                         val ->
                                 val.getLeft()
                                         .equals(
-                                                GeneCentricField.Sort.accession_id
-                                                        .getSolrFieldName()))) {
+                                                UniProtSearchFields.GENECENTRIC
+                                                        .getSortFieldFor(ACCESSION_ID)
+                                                        .getName()))) {
             return fieldSortPairs;
         } else {
             List<Pair<String, Sort.Direction>> newFieldSortPairs = new ArrayList<>();
```

Comment:
The problem with using Pair is that it is not very obvious what is in the left and what is in the right. No need to make any change. :-)

Reply:
Agreed. I would think we should raise a ticket for this and put it on the backlog